lib/colonel_kurtz/version.rb

```diff
@@ -3,6 +3,6 @@ module ColonelKurtz
   VERSION = [
     0,    # major
     0,    # minor
-    2     # patch
+    3     # patch
```

Comment:
Seems like this is a minor bump, not patch bump. It's not a bug fix, it's a change to how that object works. [semver](http://semver.org/)